qc/collect_hs_metrics.cwl

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "collect HS metrics"
+baseCommand: ["/usr/bin/java", "-Xmx16g", "-jar", "/usr/picard/picard.jar", "CollectHsMetrics"]
+arguments:
+    ["O=", { valueFrom: $(runtime.outdir)/HsMetrics.txt }]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "registry.gsc.wustl.edu/genome/picard-2.4.1-r:2"
+inputs:
+    bam:
+        type: File
+        inputBinding:
+            prefix: "I="
+        secondaryFiles: [.bai]
+    reference:
+        type: File
+        inputBinding:
+            prefix: "R="
+        secondaryFiles: [.fai]
+    bait_intervals:
+        type: File
+        inputBinding:
+            prefix: "BAIT_INTERVALS="
+    target_intervals:
+        type: File
+        inputBinding:
+            prefix: "TARGET_INTERVALS="
+outputs:
```

Comment:
I think either way is possible. The boolean arg option would be more user friendly IMHO. If the option is picked we can set PER_TARGET_COVERAGE to a hardcoded file.